joomla/joomla-cms
# Ensure we still have valid users to batch reset password

To test:
Select the user you are currently logged in with only and batch reset their password (with a value of yes or no).

Before the patch you'll get a mysql warning. After the patch you'll get a notice that no users have been selected.

This is because you cannot reset the password of the currently logged in user so suddenly you are trying to batch reset no users.
